src/api/storage-pools.tsx

```diff
@@ -72,6 +72,13 @@ export const createPool = (
   });
 };
 
+const clusterPool = (pool: Partial<LxdStoragePool>) => {
```

Comment:
This name doesn't make it very clear what the function does. Maybe something more specific like `getPoolForCluster`?

Reply:
I was struggling to find a good name. Will go with your suggestion, that is def. better.